### DISCUSSION TITLE
Irrelevant feed + too many big repos

### DISCUSSION THREAD

**Discussion by Clepalitto** [community/community#19263] [Other Feature Feedback, Questions, & Ideas] [answered]
I don't know if this is just me, but my "For you" feed indicates repos that are irrelevant.
For instance, if I checked out a repo once, then its topic would be recommended everyday for a month, even though I don't work on this topic anymore.

Also, I often see repos owned by big companies in the feed. It might also be interesting to recommend more small repos that haved needed a lot of work from its owner.

**Comment by melvincwng** [comment, marked as answer]
@Clepalitto, try following new developers and starring new repositories, every once in a while, to "spice up" your feed in the For You page.

Hope this helps 🙏

**Reply by melvincwng** [reply]
@Clepalitto, for more information regarding the For You page, please refer to [this page](https://github.blog/2022-03-22-improving-your-github-feed/) in Github Blog.

If you find this reply helpful, kindly 'Mark it as answer' so that others know this issue has been addressed.

Take care, and have a wonderful day ahead 😊

**Reply by Codel1417** [reply]
This doesn't change anything on my feed

**Comment by Clepalitto** [comment]
Thanks a lot ! 😊

**Reply by melvincwng** [reply]
@Clepalitto, welcome! Glad to have helped 👍🏻

**Comment by darkweak** [comment]
I don't like this feature, I would go back a year ago!

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
